src/udiskslinuxfilesystem.c

```diff
@@ -350,24 +350,26 @@ udisks_linux_filesystem_update (UDisksLinuxFilesystem  *filesystem,
 
 /* ---------------------------------------------------------------------------------------------------- */
 
+/* TODO: port to libblockdev */
```

Comment:
I am not sure if it make sense to add this particular list to libblockdev. We have the [proposed new API for filesystem features](https://github.com/storaged-project/libblockdev/pull/621) that will cover things like mkfs options, labeling support etc., but the list of filesystem we allow to mount looks quite udisks-specific to me to move it to libblockdev.

Reply:
Another perspective - if _mounting_ is considered a stateful concept and libblockdev is not being mount-aware (except of the actual mount call) by purpose, then it makes sense to keep it within udisks.

Either way, we should guarantee that all filesystems that udisks (through libblockdev) is able to create are always included in the list of filesystems udisks is able to mount. Without manually keeping the lists in sync.

Reply:
Anyway, I've removed the comments and added some more filesystems to the list, for the time being.